website/src/pages/index.tsx

```diff
@@ -365,7 +365,7 @@ function EnterpriseReady() {
                 <h2 className="text-gray-900 dark:text-gray-100 text-lg title-font font-medium mb-2">Code signing</h2>
 
                 <p className="leading-relaxed text-base">
-                  macOS binaries are digitally signed (Windows certification is in)
+                  Digitally signed binaries available across all leading Operating Systems
```

Comment:
```suggestion
                  Digitally signed binaries available on Windows and macOS
```

Reply:
Yeah, thought that could be the case. Thanks for the tweak.